schema/catvrs/catvrs-source.yaml

```diff
@@ -13,296 +13,254 @@ namespaces:
-      `Allele <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#variation>` that is representative of a collection
-      of congruent Protein Alleles that share the same altered codon(s).
-    properties:
-      type:
-        extends: type
-        const: ProteinSequenceConsequence
-        default: ProteinSequenceConsequence
-        description: 'MUST be "ProteinSequenceConsequence"'
-      definingContext:
-        oneOf:
-          - $refCurie: vrs:Allele
-          - $refCurie: gks.common:IRI
+          A textual description of the domain of variation that should match the categorical 
+          variation entity.
+      label:
+        extends: label
         description: >-
-          The `VRS Allele <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#allele>`_
-          object that is congruent with (projects to the same codons) as alleles on other protein reference 
-          sequences.
+            A primary label for the categorical variation. This required property should provide a 
+            short and descriptive textual representation of the concept.
     required:
-      - definingContext
+      - label
 
-  CanonicalAllele:
+  
+  definingContexConstraint:
     maturity: draft
-    inherits: CategoricalVariation
-    description: >-
-      A canonical allele is defined by an `Allele <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#variation>` 
-      that is representative of a collection of congruent Alleles, each of which depict the same nucleic acid 
-      change on different underlying reference sequences. Congruent representations of an Allele often exist
-      across different genome assemblies and associated cDNA transcript representations.
-    type: object
+    inherits: Constraint
+    description:
+      The location or location-state, congruent with other reference sequences, about which categorical variation is being described.
     properties:
-      type:
-        extends: type
-        const: CanonicalAllele
-        default: CanonicalAllele
-        description: 'MUST be "CanonicalAllele"'
+      # type:   # Do I need this part?  I don't think so?
+      #   extends: type
+      #   const: definingContext
+      #   default: definingContext
+      #   description: 'MUST be "definingContext"'
       definingContext:
         oneOf:
+          - $refCurie: vrs:Location
           - $refCurie: vrs:Allele
-          - $refCurie: gks.common:IRI
+      relations:
+        type: array
+        items:
+          # I think we need something here but I'm not sure what
         description: >-
-          The `VRS Allele <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#allele>`_
-          object that is congruent with variants on alternate reference sequences.
+          a way to represent coded relation concepts, e.g. "genome_liftover"
+      sequenceCollections:
+        type: array
+          items: $refCurie: gks.core:IRI
```

Comment:
Sequence Collections might be interesting to annotate in some way, but is not an immediate pressing concern. Also, not clear if SeqCol spec will support this use case in v1.

```suggestion
```

Reply:
Agreed. less is more to get things rolling. I also concur that this could be interesting to see play out down the road with the SeqCol folks.